airbyte-integrations/connectors/source-hubspot/source_hubspot/client.py

```diff
@@ -102,7 +102,7 @@ def health_check(self) -> Tuple[bool, str]:
 
         try:
             _ = self._apis["contacts"].properties
-        except HubspotError as error:
+        except HTTPError as error:
```

Comment:
To discuss: what's the purpose of HubspotError hierarchy then?

Reply:
to distinct API errors from others